ParabolInc/parabol
# fix: fix usage stats charts on small amount of data

Fixes #7817

Now:

<img width="1024" alt="Screenshot 2023-03-07 at 17 28 47" src="https://user-images.githubusercontent.com/466991/223382966-b80e20ca-e2af-4899-972a-c51b15557e0b.png">

Before:

<img width="1022" alt="image" src="https://user-images.githubusercontent.com/466991/223383271-55700bca-9cb3-4613-9765-e252fd074d17.png">


Also fixed axis labels for members and teams charts

<img width="1014" alt="image" src="https://user-images.githubusercontent.com/466991/224671021-72b54ab7-6189-4214-9d7f-8f16ff8714c0.png">

How to test:

- Create a fresh org
- Run and complete some meetings
- Enable usage stats
- See chars are working fine
